src/trials.jl

```diff
@@ -313,36 +340,151 @@ Base.show(io::IO, t::TrialJudgement) = _show(io, t)
+
+    print(io, "\n", pad, " Range ")
+    printstyled(io, "("; color=:light_black)
+    printstyled(io, "min"; color=:cyan, bold=true)
+    print(io, " … ")
+    printstyled(io, "max"; color=:magenta)
+    printstyled(io, "):  "; color=:light_black)
+    printstyled(io, lpad(mintime, lmaxtimewidth); color=:cyan, bold=true)
+    print(io, " … ")
+    printstyled(io, lpad(maxtime, rmaxtimewidth); color=:magenta)
+    print(io, "  ")
+    printstyled(io, "┊"; color=:light_black)
+    print(io, " GC ")
+    printstyled(io, "("; color=:light_black)
+    print(io, "min … max")
+    printstyled(io, "): "; color=:light_black)
+    print(io, lpad(mingc, lmaxgcwidth), " … ", lpad(maxgc, rmaxgcwidth))
+
+    print(io, "\n", pad, " Time  ")
+    printstyled(io, "("; color=:light_black)
+    printstyled(io, "median"; color=:blue, bold=true)
+    printstyled(io, "):     "; color=:light_black)
+    printstyled(io, lpad(medtime, lmaxtimewidth), rpad(" ", rmaxtimewidth + 5); color=:blue, bold=true)
+    printstyled(io, "┊"; color=:light_black)
+    print(io, " GC ")
+    printstyled(io, "("; color=:light_black)
+    print(io, "median")
+    printstyled(io, "):    "; color=:light_black)
+    print(io, lpad(medgc, lmaxgcwidth))
+
+    print(io, "\n", pad, " Time  ")
+    printstyled(io, "("; color=:light_black)
+    printstyled(io, "mean"; color=:green, bold=true)
+    print(io, " ± ")
+    printstyled(io, "σ"; color=:green)
+    printstyled(io, "):   "; color=:light_black)
+    printstyled(io, lpad(avgtime, lmaxtimewidth); color=:green, bold=true)
+    print(io, " ± ")
+    printstyled(io, lpad(stdtime, rmaxtimewidth); color=:green)
+    print(io, "  ")
+    printstyled(io, "┊"; color=:light_black)
+    print(io, " GC ")
+    printstyled(io, "("; color=:light_black)
+    print(io, "mean ± σ")
+    printstyled(io, "):  "; color=:light_black)
+    print(io, lpad(avggc, lmaxgcwidth), " ± ", lpad(stdgc, rmaxgcwidth))
+
+    # Histogram
+
+    histquantile = 0.99
+    histheight = 2
+    histwidth = 42 + lmaxtimewidth + rmaxtimewidth
+
+    histtimes = t.times[1:round(Int, histquantile*end)]
+    # 47 = the 'base' width such that the histogram doesn't go past the statistics text
+    bins, logbins = bindata(histtimes, histwidth - 1), false
+    append!(bins, [1, floor((1-histquantile) * length(t.times))])
+    # if median size of (bins with >10% average data/bin) is less than 5% of max bin size, log the bin sizes
+    if median(filter(b -> b > 0.1 * length(t.times) / histwidth, bins)) / maximum(bins) < 0.05
+        bins, logbins = log.(1 .+ bins), true
+    end
+    hist = asciihist(bins, histheight)
+    hist[end,end-1] = ' '
+    maxbin = maximum(bins)
+
+    delta1 = (histtimes[end] - histtimes[1]) / (histwidth - 1)
+    medpos = 1 + round(Int, (histtimes[length(histtimes) ÷ 2] - histtimes[1]) / delta1)
+    avgpos = 1 + round(Int, (mean(histtimes) - histtimes[1]) / delta1)
```

Comment:
As mentioned earlier, the columns highlighted on the histogram are the median and the mean of the filtered times (the bottom 99% of times), which means that they may not match up with the text above, which shows the reported median and mean of all the times.

You can see this in my example from earlier:
![2021-07-09-093910_790x292_scrot](https://user-images.githubusercontent.com/6000761/125051403-d51d8880-e09a-11eb-8c9c-ddd01e5b843e.png)

Here, the median column clearly does not contain the median time.

Looks like a simple enough fix. If I have time today I will make a PR.

Reply:
Cool, a fix PR would be nice :) Though since it's a log y-axis there I wouldn't be surprised if nothing changed.

Reply:
It's the position on the x axis that's wrong. Unless I'm badly misunderstanding something it's definitely highlighting the wrong column

Image showing where I think it should go:

![2021-07-09-093910_790x292_scrot-2](https://user-images.githubusercontent.com/6000761/125064879-0735e700-e0a9-11eb-8319-dbc5fe11a7d4.png)

Reply:
Yea, that's definitely not what it should be. The leftmost bar is the tallest, and so I'd expect the median to be there. Since this is a log-y being slightly taller is exponentially more points in that bin. The reason why I introduced a log mode at all is so that you could see if any other times where being hit more than others, as otherwise that single bar will completely dominate the plot.

Reply:
Ah, ooops - I misread the numbers above ... somehow. I think you're completely right. Cary on :sweat_smile: